conf/gconf/cluster.conf.sample

```diff
@@ -0,0 +1,258 @@
+  name: cortx-cluster
+  storage_set_count: 1
+  storage_set:
+  - name: storage-set-1
+    durability:
+      dix:
+        data: '1'
+        parity: '0'
+        spare: '0'
+      sns:
+        data: '1'
+        parity: '0'
+        spare: '0'
+    nodes:
+    - 9995f539f4f770e2a3fe9e2e615c32a8
+    - 1115f539f4f770e2a3fe9e2e615c32a8
+    - aaa120a9e051d103c164f605615c32a4
+    - bbb340f79047df9bb52fa460615c32a5
+    - ccc8700fe6797ed532e311b0615c32a7
+    - ddd120a9e051d103c164f605615c32a4
+    - eee340f79047df9bb52fa460615c32a5
+    - fff8700fe6797ed532e311b0615c32a7  
+
+node:
+  1115f539f4f770e2a3fe9e2e615c32a8:
+    type: ha_node
+    cluster_id: 0007ec45379e36d9fa089a3d615c32a3
+    components:
+    num_components: 2
+    - name: utils
+      version: 2.0.0-33
+    - name: ha
+      version: 2.0.0-47
+    hostname: ha-node
+    name: ha-node
+
+    provisioning:
+      phase: deployment
+      status: success
+      version: 2.0.0-576
+    storage_set: storage-set-1
+
+  9995f539f4f770e2a3fe9e2e615c32a8:
+    type: control_node
+    hostname: control-node
+    name: control-node
+    cluster_id: 0007ec45379e36d9fa089a3d615c32a3
+    num_components: 4
+    components:
+    - name: utils
+      version: 2.0.0-33
+    - name: motr
+      version: 2.0.0-33
+    - name: csm
+      services:
+      - agent
+      version: 2.0.0-21
+    provisioning:
+      phase: deployment
+      status: success
+      version: 2.0.0-576
+    storage_set: storage-set-1
+
+  aaa120a9e051d103c164f605615c32a4:
+    type: data_node
+    hostname: data-node1
+    name: data-node1
+    cluster_id: 0007ec45379e36d9fa089a3d615c32a3
+    num_components: 3
+    components:
+    - name: utils
+      version: 2.0.0-33
+    - name: motr
+      services:
+      - io
+      version: 2.0.0-33
+    - name: hare
+      version: 2.0.0-47
+    storage:
+      cvg:
+      - devices:
+          data:
+          - /dev/sdc
+          - /dev/sdd
+          metadata:
+          - /dev/sdb
+        name: cvg-01
+        type: ios
+      - devices:
+          data:
+          - /dev/sdf
+          - /dev/sdg
+          metadata:
+          - /dev/sde
+        name: cvg-02
+        type: ios
+      cvg_count: 2
+    provisioning:
+      phase: deployment
+      status: success
+      version: 2.0.0-576
+    storage_set: storage-set-1
+
+  ddd120a9e051d103c164f605615c32a4:
+    cluster_id: 0007ec45379e36d9fa089a3d615c32a3
+    components:
+    - name: utils
+      version: 2.0.0-33
+    - name: hare
+      version: 2.0.0-47
+    hostname: server-node1
+    name: server-node1
+    num_components: 3
+    storage_set: storage-set-1
+    type: server_node
+    
+  eee340f79047df9bb52fa460615c32a5:
+    type: server_node
+    hostname: server-node2
+    name: server-node2
+    cluster_id: 0007ec45379e36d9fa089a3d615c32a3
+    num_components: 3
+    components:
+    - name: utils
+      version: 2.0.0-33
+    - name: hare
+      version: 2.0.0-47
+    - name: rgw_s3
```

Comment:
@cdeshmukh, @sachitanands 
As we discussed here component name should be '**rgw**' and no need to define rgw_s3 and rgw_admin separately.
Please confirm.

Reply:
agree